lib/MetaCPAN/Server/Controller/Changes.pm

```diff
@@ -65,4 +65,41 @@ sub all : Chained('index') : PathPart('') : Args(0) {
     $c->detach('not_found');
 }
 
+sub by_releases : Path('by_releases') : Args(0) {
+    my ( $self, $c ) = @_;
+
+    # ArrayRef[ Dict[ author => Str, name => Str ] ]
+    my $arg = $c->read_param("releases");
```

Comment:
Most other places we are accepting release names, we accept them in `AUTHOR/Dist-Name-v1.234` form.  I wonder if we should do the same here.  This would also have the advantage of being able to use URL or form parameters, and not forcing the use of JSON.

Reply:
Thank you very much for the pointer. This is one of the places I'm not feeling confident about due to the lack of knowig what's conventional or not.

Assuming `,` character is invalid in release names... I'm guessing it is OK to use a CSV form in the URL to represent a list of release names, like this:

    /feed/recent/?releases=AUTHOR1/DistName-v1.23,AUTHOR2/DistName-v1.24,AUTHOR3/DistName-v1.24

Would that fit the convention here ?

Reply:
I was thinking like `/changes/by_releases?release=AUTHOR1/DistName-v1.23&release=AUTHOR2/DistName-v1.24`.

Reply:
The `by_releases` method is changed according to what's described by @haarg -- I've also updated the description text to match the implementation.